packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -104,6 +104,8 @@ export interface Props {
   onToggle?: (isOpen: boolean) => void;
   /** Additional class name for the root element */
   className?: string;
+  /** Element child component */
+  children?: React.ReactNode;
```

Comment:
`FunctionComponent` type already has children defined, so this addition here shouldn't be necessary. Instead you could pass the children prop down to Collapse:
```
    <Collapse
      isOpen={open}
      collapsible
      {...otherProps}
      onToggle={() => {
        setOpen(!open);
        if (onToggle) {
          onToggle(!open);
        }
      }}
    >
      {children}
    </Collapse>
```

Reply:
Ooooo interesting maybe I see this error because it's FunctionComponent<Props> instead of FunctionComponent<PropsWithChildren>? I can update to `FunctionComponent<PropsWithChildren>` if you prefer that.
![Screen Shot 2022-10-13 at 9 08 37 AM](https://user-images.githubusercontent.com/114438185/195651839-31fbe5ea-e2fe-4f49-92e9-1fccfc3d0709.png)
![Screen Shot 2022-10-13 at 9 09 41 AM](https://user-images.githubusercontent.com/114438185/195651873-48a514bd-149c-4ea6-9f3b-d88b15f78d60.png)

Reply:
If you check the type definition for `FunctionComponent`, it already extends `PropsWithChildren`, so passing it additionally isn't necessary. For the same reason it's not necessary to define an extra `children` prop.
```
    interface FunctionComponent<P = {}> {
        (props: PropsWithChildren<P>, context?: any): ReactElement<any, any> | null;
        propTypes?: WeakValidationMap<P> | undefined;
        contextTypes?: ValidationMap<any> | undefined;
        defaultProps?: Partial<P> | undefined;
        displayName?: string | undefined;
    }
```
We have existing code that passes `children` to `Collapse` component and works without problems, so this might be an issue with your local setup.

Reply:
Okay, I'll close it for now and just keep our local collapse component, while investigating our ts setup. TY!